tests/blackbox/common/functions.ts

```diff
@@ -755,12 +768,18 @@ export async function CreatePolicy(vendor: Vendor, options: OptionsCreatePolicy)
 export type OptionsCreatePermission = {
 	role: keyof typeof ROLE;
 	permission: Omit<Partial<Permission>, 'id' | 'role' | 'system'>;
-	policy?: string;
+} & OptionsCreatePermissionPolicy;
+
+export type OptionsCreatePermissionPolicy = {
+	policyId?: string;
 	policyName?: string;
 };
 
-export async function CreatePermission(vendor: Vendor, options: OptionsCreatePermission) {
-	let policyId = options.policy;
+export async function CreatePermissionWithPolicy(
```

Comment:
Not too sure about the renaming, `CreatePermission` should be able to account for the creation with/without policy.

Reply:
I created a new function which _only_ creates a new permission and this I called `CreatePermission`. So I had to find a new name for the function which creates a permission and add the permission to a policy.

do you have an alternative name? it seems that the permission is added to a policy either way. It either creates a new policy or uses the provided policy id